src/test/interfaces_tests.cpp

```diff
@@ -98,7 +98,7 @@ BOOST_AUTO_TEST_CASE(findCommonAncestor)
     auto* orig_tip = active.Tip();
     for (int i = 0; i < 10; ++i) {
         BlockValidationState state;
-        m_node.chainman->ActiveChainstate().InvalidateBlock(state, Params(), active.Tip());
+        m_node.chainman->ActiveChainstate().InvalidateBlock(state, active.Tip());
```

Comment:
It seems to me that `#include <chainparams.h>` is no longer needed.

Reply:
Thanks, will try to remove if I have to force push